package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "child_pty": "3.0.1",
     "default-shell": "1.0.1",
+    "electron-config": "^0.2.0",
```

Comment:
this isn't a dependency in `app/package.json` because it has a dynamic require which makes it not work with webpack